Stream notebook loading progressively (#593)
jupyterlab/jupyter-collaboration: docs/source/configuration.md

# If None, the document will be kept in memory forever.
jupyter lab --YDocExtension.document_cleanup_delay=100

# Whether documents loaded from disk should be streamed progressively (default: True).
jupyter lab --YDocExtension.document_load_progressively=False

> # Delay document outputs larger than this size in MB during progressive loading (default: 100).
> # If None, outputs are loaded with their content.
> jupyter lab --YDocExtension.document_output_delay_threshold_mb=50

Reviewer: ```suggestion
# Defer notebook outputs larger than this size in MB during progressive loading (default: 100).
# If None, outputs are loaded with the inputs.
jupyter lab --YDocExtension.notebook_output_defer_threshold_mb=50
```

Reviewer: Ah since jupyter-ydoc already uses "delay" we should keep that word.
